ambari-logsearch-logfeeder/src/main/scripts/logfeeder.sh

```diff
@@ -80,8 +80,12 @@ else
   LOGFEEDER_GC_LOGFILE="$LOG_PATH_WITHOUT_SLASH/$LOGFEEDER_GC_LOGFILE"
 fi
 
-LOGFEEDER_GC_OPTS="-XX:+PrintGCDetails -XX:+PrintGCDateStamps -Xloggc:$LOGFEEDER_GC_LOGFILE"
-#LOGFEEDER_GC_OPTS="-Xlog:gc*:file=$LOGFEEDER_GC_LOGFILE:time" #TODO: check java version, use this if not JDK8 is used
+java_version=$($JVM -version 2>&1 | grep 'java version' | cut -d'"' -f2 | cut -d'.' -f2)
```

Comment:
Grepping for `java version` may fail to match if run with OpenJDK.  See apache/ambari@b6309a8160151387510875a6d27c340da58eb626#diff-f442ddfc811d70249788d702e79e6813R1275 for working regex (so far).

Reply:
I'm not sure is it valid for jdk 1.8 only, so for example for other inputs (which is not 1.8) we won't care with the response, and then it will use the other gc settings